source/guide_grocy.rst

```diff
@@ -0,0 +1,139 @@
+.. highlight:: console
+
+.. author:: Jonas <https://github.com/jfowl>
+
+.. tag:: lang-php
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/grocy.svg
+      :align: center
+
+####
+Grocy
+####
+
+.. tag_list::
+
+Grocy is a web-based self-hosted groceries & household management solution for your home.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * Git_
```

Comment:
Currently you don't use any git command in your guide. So I don't know why it is a need-to-know.

There is a way, but I don't think this is the way the target users should use it as it requires some other steps.

Grocy README.md:
> Alternatively clone this repository (the release branch always references the latest released version, or checkout the latest tagged revision) and install Composer and Yarn dependencies manually.

Reply:
Fixed.